package.json

```diff
@@ -25,7 +25,7 @@
     "ripple-lib-transactionparser": "^0.3.2",
     "ripple-wallet-generator": "^1.0.3",
     "simple-asyncify": "^0.1.0",
-    "sjcl": "^1.0.3",
+    "sjcl-extended": "ripple/sjcl-extended#1.0.3",
```

Comment:
It's funny that npm has a built-in syntax just for GitHub: https://docs.npmjs.com/files/package.json#github-urls